src/components/Body.js

```diff
@@ -0,0 +1,42 @@
+import "./Body.css";
+import searchImg from "../assets/Search.svg";
+import CardList from "./CardList.js";
+import { getCards } from "../api.js";
+import { useEffect, useState } from "react";
+
+function Body() {
+    const [items, setItems] = useState([]);
+
+    const handleLoad = async () => {
+        let result;
+        try {
+            result = await getCards();
+        } catch (error) {
+            console.error(error);
+            return;
+        }
+
+        const { folder } = result;
+        setItems(folder.links);
+    };
```

Comment:
변수를 선언할 때 let이 꼭 필요한 경우도 있기는 한데요..!
대부분은 const로 가능해서 왠만하면 const로 사용하시는 걸 의식적으로 해보시면 좋을 것 같아요!
```suggestion
    const handleLoad = async () => {
        try {
            const result = await getCards();
            const { folder } = result;
            
            setItems(folder.links);
        } catch (error) {
            console.error(error);
        }
    };
```

Reply:
현재는 에러가 발생했을 때(catch) 콘솔에 에러를 출력해주고 있는데요~
일반 사용자가 웹사이트를 이용할 때 콘솔을 확인하면서 이용할 일이 없으니까 실제로 에러처리는 다른식으로 표현이 되어야 하고 콘솔에 에러를 출력하는 건 개발자가 확인하는 용도라고 생각해주시면 좋을 것 같아요!

에러 처리를 위한 방법으로는 이런것들이 있을 것 같아요~
- 에러가 발생했을 때 상태값을(isError) 변경시켜서 화면에 에러 텍스트를 보여준다.
  - 조금 더 고려하면 `다시 시도하기`와 같은 버튼을 클릭했을 때 api를 다시 호출하도록 한다.
- 에러가 발생했을 때 화면에 팝업을 띄운다.